tags/3.0/_index.md

```diff
@@ -22,22 +22,25 @@ existing custom tags with Jakarta Standard Tag Library tags.
 
 The plan for the release is to clean up open issues:
 * General specification document cleanup.
-* Clean up issues found during the Jakarta EE9 review ballot process: update if possible the tagdocs to include EFSL, Update tlddoc generation if possible to jakarta.
+* Clean up issues found during the Jakarta EE9 review ballot process
+* Update tagdocs to include EFSL
+* Update tlddoc generation to jakarta.
 * Remove any deprecated methods from the API if they exist.
 * Rename java.sun.com / xmlns.jcp.org taglib URIs to new jakarta.tags.\* URNs. [Issue 144](https://github.com/eclipse-ee4j/jstl-api/issues/144)
     * This alone should require a major release.
 * Other small clean up issues
```

Comment:
@ivargrimstad 
Should I update this section to list a `Release Summary` (and include list of changes) or should I just remove entire section? 

Pages 3.1 and a few others, for example, don't include a Plan or other summary.

https://deploy-preview-435--jakartaee-specifications.netlify.app/specifications/pages/3.1/